beanquery/shell_test.py

```diff
@@ -18,7 +18,7 @@
 def setup_module():
     example_filename = path.join(beanquery_test_utils.find_repository_root(__file__),
                                  'examples', 'example.beancount')
-    global entries, errors, options_map  # pylint: disable=invalid-name
+    global entries, errors, options_map  # pylint: disable=global-statement
```

Comment:
I was getting following error:

```
W0603: Using the global statement (global-statement)
```

For now I just disabled it, but if you prefer I can try to fix this (I assume I would need to add some kind of singleton, or read more the best way how to do one time initialization in python tests.

Reply:
See #18